APIs/StreamCompatibilityManagementAPI.raml

```diff
@@ -293,6 +293,14 @@ mediaType: application/json
             403:
         put:
           description: Set Base EDID
+          queryParameters:
+            adjust_to_caps:
+              description: >
+                If true, the Input is allowed to adjust Base EDID to internal capabilities.
+                If omitted for an Input that supports it, the previous value is preserved.
```

Comment:
I don't 100% understand either sentence.

If true, the Input is allowed to adjust... but doesn't have to?

If omitted (or false?), the previous value is preserved... Which previous value? The previous `adjust_to_caps` value, or the previous Base EDID? Wouldn't that be an error?

Reply:
Removed the functional description from the RAML spec. 49a62d311d515695754ea0022b17d3cb7acda7c4